docs/content/usage/access-secret-cfgmap-in-function.en.md

```diff
@@ -101,7 +101,7 @@ $ fission env create --name python --image fission/python-env
 # create function named "leaker"
 $ fission fn create --name leaker --env python --code leaker.py --secret my-secret --configmap my-configmap
 ```
-
+The fission CLI doesnt support providing two configmaps for a function for now, but it can be achieved through the fission spec. We can provide two configmaps for a function in the fission spec yaml.
```

Comment:
Nice, thanks.  Could you link to the "source code organization" page for the "fission spec" text?  It's at `/usage/developer-workflow/`

Reply:
sure I will do that.